riff-raff/app/restrictions/RestrictionConfig.scala

```diff
@@ -0,0 +1,32 @@
+package restrictions
+
+import java.util.UUID
+
+import org.joda.time.DateTime
+
+/**
+  * A configuration that restricts which deploys are allowed
+  * @param id Unique ID for CRUD storage and retrival
+  * @param projectName The project name that this applies to
+  * @param stage The stage that this applies to (this can be a regex, but if no special regex characters are found then
```

Comment:
Would it be better to have this be a regex type field? A string literal is a subset of a the regex type and this would move the "if no special character" logic to the edge.